cdk/src/open-data-platform/data-plane/data-import/utils/lambda-function-factory.ts

```diff
@@ -35,3 +36,29 @@ export const lambdaFactory = (
     },
   });
 };
+
+/**
+ * Produces a lambda function that requires db access to import data.
+ * @param scope: the construct under which to provision resource.
+ * @param props: schema identifiers passed to the function when writing data.
+ * @param id: ID to assign resource.
+ */
+export const apiLambdaFactory = (
```

Comment:
weirdly not seeing code highlighted here - do you know why that is? is this commented out?

Reply:
Hmm no but the cdk deploy works so it should be okay